certbot-auto

```diff
@@ -965,18 +956,10 @@ if [ "$1" = "--le-auto-phase2" ]; then
     say "Creating virtual environment..."
     DeterminePythonVersion
     rm -rf "$VENV_PATH"
-    if [ "$PYVER" -le 27 ]; then
```

Comment:
I don't think these lines should be changed because we still need the `virtualenv` version for Python 2.7 support.